src/ZODB/interfaces.py

```diff
@@ -1209,8 +1218,8 @@ def release():
     def poll_invalidations():
         """Poll the storage for external changes.
 
-        Returns either a sequence of OIDs that have changed, or None.  When a
-        sequence is returned, the corresponding objects should be removed
+        Returns either None, or a sequence of OIDs that have created and changed.
```

Comment:
"OIDs that have __been__ created __or__ changed."

Reply:
ok, but added "the full set" and reference to "previous call to poll invalidations":

_"Returns either None, or a sequence of OIDs **with the full set of objects** that have been created or changed **since previous call to poll_invalidations**."_